### DISCUSSION TITLE
Unicode variants of operators shouldn't be an 'implemenation option'?

### DISCUSSION THREAD

**Discussion by moon-chilled** [odin-lang/Odin#954] [General]
The [spec](https://odin-lang.org/ref/spec/) says:

> Implementation option: A compiler may allow the following character sequences as aliases for other operators, punctuation, and keywords:
> `'≠' (U+2260) alias for '!='`
> `'≤' (U+2260) alias for '<='`
> `'≥' (U+2260) alias for '>='`
> `'∈' (U+2260) alias for 'in'`
> `'∉' (U+2260) alias for 'notin'`

Obviously certain things should be implementation-defined, but imo unicode operators don't make sense as such; implementation-defined only really makes sense for things that vary across platforms.  Personally, I would remove them entirely, but either way they I think they should either be there, or not.

**Comment by gingerBill** [comment]
This just added because I could rather than it be actually a good thing. It is subject to be removed.
